### PR TITLE
adding commands ps and collect to docs

### LINKED ISSUE #17220: missing dotnet-counters commands
this page is missing the following commands:
- collect
- ps

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a4dadb56-569e-b390-b118-57c3b0225c13
* Version Independent ID: 50cf6a7e-e95e-a29b-67fd-26747f4440c1
* Content: [dotnet-counters - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-counters#feedback)
* Content Source: [docs/core/diagnostics/dotnet-counters.md](https://github.com/dotnet/docs/blob/master/docs/core/diagnostics/dotnet-counters.md)
* Product: **dotnet-core**
* GitHub Login: @sdmaclea
* Microsoft Alias: **stmaclea**

### PR DESCRIPTION
**Pull request by eddynaka**
Fixes #17220

### CODE REVIEW

**Comment by sdmaclea** [comment]
@sywhang When were these new subcommands introduced?  Do we need any applies to info update?

**Review by sdmaclea — APPROVED** [review]
LGTM except applies to question

**Comment by sywhang** [comment]
@sdmaclea I don't remember the exact version, but since it's relatively easy for customers to just update to the latest version I think it's fine to leave out applies to info?

**Review by sywhang — APPROVED** [review]

File: docs/core/diagnostics/dotnet-counters.md [hunk 1]
```diff
@@ -39,8 +39,53 @@ dotnet-counters [-h|--help] [--version] <command>
```
Comment (mairaw) [inline docs/core/diagnostics/dotnet-counters.md]: Given the nature of the changes on this article, please update the ms.date as well on line 4 to 02/26/2020

Comment (eddynaka) [reply docs/core/diagnostics/dotnet-counters.md]: done! thanks for the tip!

**Review by BillWagner — APPROVED** [review]
LGTM. I'll :shipit:

**Pull request merged**